roles/ceph-client/defaults/main.yml

```diff
@@ -1,18 +0,0 @@
----
```

Comment:
Thanks for your comments @andrewschoen .

The idea is to have `ceph-defaults` handling only defaults variables and `ceph-config` for stuff like setting `fsid`. Do you think merging these two roles would be better?

Don't you think it could be cool to keep the default behavior which is including `ceph-common` or `ceph-docker-common` from other roles but having the possibility to dismiss this it as well ([see here](https://github.com/ceph/ceph-ansible/pull/1710/files#diff-27c326e40da68063aa62154f73c13a19R14)) so we can set dependencies in `site.yml` as you describe above?

Even if the CI seems to be happy, this PR is not finished at all, so, please let me know if you have any thoughts or suggestions.